src/config.js

```diff
@@ -0,0 +1,391 @@
+   * config based on a JSON since Caravan is the assumed format of accepted JSON
+   * @param {string} jsonString - valid config json string from Caravan export
+   * @returns {MultisigWalletConfig} new instance of a MultisigWalletConfig
+   */
+  static fromCaravanConfig(jsonString) {
+    return MultisigWalletConfig.fromJSON(jsonString);
+  }
+
+  /**
+   * @description Takes a coldcard config string and creates
+   * a new instance of a MultisigWalletConfig.
+   * 
+   * NOTE for ColdCard: only the root xfp of the signing device is required to be correct, but we
+   * recommend only setting up the multisig wallet on the Coldcard with complete xfp
+   * information.
+   * 
+   * This is an example Coldcard config file from
+   * https://coldcardwallet.com/docs/multisig
+   * 
+   * # Coldcard Multisig setup file (exported from 4369050F)
+   * #
+   * Name: MeMyself
+   * Policy: 2 of 4
+   * Derivation: m/45'
+   * Format: P2WSH
+   * 
+   * D0CFA66B: tpubD9429UXFGCTKJ9NdiNK4rC5...DdP9
+   * 8E697B74: tpubD97nVL37v5tWyMf9ofh5rzn...XgSc
+   * BE26B07B: tpubD9ArfXowvGHnuECKdGXVKDM...FxPa
+   * 4369050F: tpubD8NXmKsmWp3a3DXhbihAYbY...9C8n
+   * 
+   * @param {string} configString valid coldcard config string
+   * @returns {MultisigWalletConfig} new instance of MultisigWalletConfig
+   * @example
+   * import { MultisigWalletConfig } from "unchained-wallets";
+   * 
+   * const config = MultisigWalletConfig.fromColdCardConfig(ccFileData);
+   * console.log(JSON.parse(config.toJSON()))
+   * 
+   * {
+   *    name: MeMyself,
+   *    quorum: {
+   *      totalSigners: 4,
+   *      requiredSigners: 2,   
+   *    },
+   *    addressType: 'P2WSH',
+   *    network: 'testnet',
+   *    extendedPublicKeys: [
+   *      {
+   *        xfp: 'D0CFA66B',
+   *        xpub: 'tpubD9429UXFGCTKJ9NdiNK4rC5...DdP9'
+   *      },
+   *      {
+   *        xfp: '8E697B74',
+   *        xpub: 'tpubD97nVL37v5tWyMf9ofh5rzn...XgSc',
+   *      },
+   *      {
+   *        xfp: 'BE26B07B',
+   *        xpub: 'tpubD9ArfXowvGHnuECKdGXVKDM...FxPa',
+   *      },
+   *      {
+   *        xfp: '4369050F',
+   *        xpub: 'tpubD8NXmKsmWp3a3DXhbihAYbY...9C8n',
+   *      }
+   *    ]
+   * }
+   */
+  static fromColdCardConfig(configString) {
+    if (typeof configString !== 'string') {
+      throw new TypeError(
+        `ColdCard config must be a string, instead received ${typeof configString}`
+      );
+    }
+    
+    const options = {};
+    // split the text by line
+    const lines = 
+      configString
+        .split('\n')
+        // formatting
+        .map(line => line.trim())
+        // ignore comments
+        .filter(line => line[0] !== '#' && line.length);
+
+    // for each line separate key and values by colon
+    lines.forEach(line => {
+      // trim white space and set to lowercase
+      const [key, value] = line.split(':').map(str => str.trim());
+      // handle recognized keys (name, format, derivation, and policy)
```

Comment:
maybe not needed atm but derivation isn't handled

Reply:
we don't write this out bc right now it's a fixed path for all xpubs, which isn't necessarily true in our use case.

Reply:
Yeah that's probably why I originally skipped (and the fixtures didn't have) it but I think it should still be worth adding now with some basic path validation.